dummy_robot/dummy_robot_bringup/launch/dummy_robot_bringup.launch.py

```diff
@@ -17,19 +17,36 @@
 from launch import LaunchDescription
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
 
 
 def generate_launch_description():
+
     pkg_share = FindPackageShare('dummy_robot_bringup').find('dummy_robot_bringup')
+
+    default_rviz_config_path = os.path.join(pkg_share, 'rviz/dummy_robot.rviz')
+    rviz_config_file = LaunchConfiguration('rviz_config_file')
+
     urdf_file = os.path.join(pkg_share, 'launch', 'single_rrbot.urdf')
+
+    rviz_config_file_arg = DeclareLaunchArgument(
+        'rviz_config_file',
+        default_value=default_rviz_config_path
+    )
+
     with open(urdf_file, 'r') as infp:
         robot_desc = infp.read()
     rsp_params = {'robot_description': robot_desc}
 
     return LaunchDescription([
+        rviz_config_file_arg,
         Node(package='dummy_map_server', executable='dummy_map_server', output='screen'),
         Node(package='robot_state_publisher', executable='robot_state_publisher',
              output='screen', parameters=[rsp_params]),
         Node(package='dummy_sensors', executable='dummy_joint_states', output='screen'),
-        Node(package='dummy_sensors', executable='dummy_laser', output='screen')
+        Node(package='dummy_sensors', executable='dummy_laser', output='screen'),
+        Node(package='rviz2', executable='rviz2', name='rviz2', output='screen',
```

Comment:
By doing this, we are creating an executable dependency between this package and `rviz2` that didn't exist before.  At the very least, we would need to declare that dependency in the package.xml file as an `exec_depend`.  However, I'm not entirely sure whether we should do that; there are currently no dependencies between the demos repository and rviz.  I'm not sure whether this will create a dependency cycle or not.  @adityapande-1995 @audrow @mjeronimo what do you think?

Reply:
I'm in favor of adding the dependency, as I think that it's better to prioritize a complete demo than fewer dependencies.

That being said, we could also include this code but comment it out with a note to add `rviz2` as a dependency if you want to use it.

Reply:
Noted. Will do.

Reply:
I'd be up for a complete demo, if it requires rviz. I can trigger full CI to check if we have a dependency cycle.